src/test/kotlin/de/bund/digitalservice/useid/refresh/RefreshControllerIntegrationTest.kt

```diff
@@ -61,9 +61,9 @@ class RefreshControllerIntegrationTest(@Autowired val webTestClient: WebTestClie
             .expectStatus()
             .is3xxRedirection
             .expectHeader()
-            // "some-refresh-address" value is defined in the test application.yaml
+            // "valid-refresh-address" value is defined in application.yaml under test
```

Comment:
Just a comment, but still..
```suggestion
            // "valid-refresh-address-1" value is defined in application.yaml under test
```